.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
vendor dir you need to add here also, right ?

Reply:
we are not using local vendor folder. but i have already added now just in case

Reply:
OK, good :)